<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addstu</class>
 <widget class="QWidget" name="addstu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="LineEdit_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Label_4">
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="LineEdit_3">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Label_2">
      <property name="text">
       <string>学号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Label">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="LineEdit_4">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LineEdit_5"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Label_5">
      <property name="text">
       <string>电话</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Label_3">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>欢迎注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
background-color: rgb(0, 155, 240);
 border-radius: 3px;
 color: rgb(255, 255, 255);
 } 
QPushButton:hover
 { 
background-color: rgb(245, 69, 255); 
}
</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
background-color: rgb(0, 155, 240);
 border-radius: 3px;
 color: rgb(255, 255, 255);
 } 
QPushButton:hover
 { 
background-color: rgb(245, 69, 255); 
}
</string>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
background-color: rgb(0, 155, 240);
 border-radius: 3px;
 color: rgb(255, 255, 255);
 } 
QPushButton:hover
 { 
background-color: rgb(245, 69, 255); 
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LineEdit</tabstop>
  <tabstop>LineEdit_2</tabstop>
  <tabstop>LineEdit_3</tabstop>
  <tabstop>LineEdit_4</tabstop>
  <tabstop>LineEdit_5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
